quickstarts/java/elasticsearch-query/logo.svg

```diff
@@ -0,0 +1,10 @@
+<svg height="-59" viewBox="86.8 87.5 404.2 80" width="2500" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This SVG doesn't render anything as far as I can tell can you check to ensure this is correct?

Reply:
You can see how it renders here: https://github.com/newrelic/newrelic-quickstarts/blob/b9bdf0797d39261bcbcfb636f1282467484d4785/quickstarts/java/elasticsearch-query/logo.svg

Reply:
apologies for that, fixed this https://github.com/hsinghkalsi/newrelic-quickstarts/blob/f607851ecf97ce7d1885415cf3045874e14af762/quickstarts/java/elasticsearch-query/logo.svg

Reply:
Thanks!